lib/pbench/server/api/__init__.py

```diff
@@ -71,7 +71,7 @@ def register_endpoints(api, app, config):
     )
     api.add_resource(
         DatasetsDetail,
-        f"{base_uri}/datasets/detail",
+        f"{base_uri}/datasets/detail/<string:name>",
```

Comment:
Wait; was I imagining it, or did you just rewrite history? Your first commit had been `dataset_name` and I suggested `dataset` which is what I'd used for metadata, for consistency. Now you have two commits in the PR and both use "name", which is confusing. I'd prefer that we make the URI mapping consistent across the APIs even though the template tag is more or less private to the API class; given that "dataset" is already in the repo, I'd prefer sticking with that. (It seemed more expressive than "name" at the time, and I don't have a better suggestion.) Besides, with my schema changes we can make this `DATASET` *type*, and the converted value will actually *be* the `Dataset` reference, not the "name" (which feeds into the idea of using the index map instead of the date range, along with authorizing against the dataset... none of which, granted, is or should be part of this PR...)

Reply:
Oh! Okay I thought you are suggesting to use name instead of dataset_name. I made the change and squashed all the changes and that rewrote the history (I guess) but yeah using dataset make sense instead of name.